### PR TITLE
remove exception and use getOrCreateSparkContext

### PR DESCRIPTION
**Pull request by Le-Zheng**

### CODE REVIEW

**Comment by Le-Zheng** [comment]
Can we remove the exception "There's existing SparkContext. Please close it first."?  
Shall we use `getOrCreateSparkContext` for  `init_orca_context`?

**Comment by Le-Zheng** [comment]
For https://github.com/intel-analytics/analytics-zoo/issues/3513

File: pyzoo/zoo/util/spark.py [hunk 1]
```diff
@@ -31,9 +31,6 @@ def __init__(self,
                  redirect_spark_log=True):
         self.spark_log_level = spark_log_level
         self.redirect_spark_log = redirect_spark_log
-        with SparkContext._lock:
-            if SparkContext._active_spark_context:
-                raise Exception("There's existing SparkContext. Please close it first.")
```
Comment (hkvision) [inline pyzoo/zoo/util/spark.py:36]: Change this to a warning?

Comment (Le-Zheng) [reply pyzoo/zoo/util/spark.py:36]: Done

**Comment by Le-Zheng** [comment]
Jenkins http://10.239.47.210:18888/view/ZOO-PR/job/ZOO-PR-Validation/5113/

File: pyzoo/zoo/util/spark.py [hunk 2]
```diff
@@ -33,7 +33,7 @@ def __init__(self,
         self.redirect_spark_log = redirect_spark_log
         with SparkContext._lock:
             if SparkContext._active_spark_context:
-                raise Exception("There's existing SparkContext. Please close it first.")
+                print("WARNING: There's existing SparkContext. Please close it first.")
```
Comment (hkvision) [inline pyzoo/zoo/util/spark.py:36]: WARNING: Note that there's an existing SparkContext. Using the existing SparkContext.

Please close it first doesn't make sense if this is a warning.

Comment (Le-Zheng) [reply pyzoo/zoo/util/spark.py]: Done

**Review by hkvision — APPROVED** [review]

File: pyzoo/zoo/util/spark.py [hunk 3]
```diff
@@ -33,7 +33,7 @@ def __init__(self,
         self.redirect_spark_log = redirect_spark_log
         with SparkContext._lock:
             if SparkContext._active_spark_context:
-                raise Exception("There's existing SparkContext. Please close it first.")
+                print("WARNING: Note that there's an existing SparkContext. Using the existing SparkContext.")
```
Comment (leonardozcm) [inline pyzoo/zoo/util/spark.py:36]: Maybe this line should be split in two?http://10.239.47.210:18888/job/ZOO-PR-Python-Spark-2.4-py37/1949//console

Comment (Le-Zheng) [reply pyzoo/zoo/util/spark.py:36]: fixed in https://github.com/intel-analytics/analytics-zoo/pull/3535

**Pull request merged**